client/src/index.jsx

```diff
@@ -14,13 +15,34 @@ const Wrapper = Styled.div`
 class App extends React.Component {
   constructor() {
     super();
+    
+    this.state = {
+      locations: [],
+      materials: []
+    }
+    this.getProductById = this.getProductById.bind(this);
   }
   
+  getProductById() {
+    let id = window.location.pathname;
```

Comment:
because of my application using react hashRouter, I think that the window.location.pathname might be different, once you have it loaded on the proxy you'll have to inspect the window.location to find out which variable you'll need to use...there definetly is one that will get you where you need to go though

Reply:
Ok! Sounds good.